app/assets/javascripts/listings/components/features-section.html.slim

```diff
@@ -11,9 +11,11 @@ accordion-heading
   .content-tile.feature-tile
     feature-row title="Neighborhood" description="{{::$ctrl.parent.listing.Neighborhood}}"
     feature-row title="Built" description="{{::$ctrl.parent.listing.Year_Built}}"
+    feature-row title="Appliances" description="{{::$ctrl.parent.listing.Appliances}}"
+    feature-row title="{{$ctrl.parent.isOwnershipListing() ? 'Services Covered by HOA Dues' : 'Services Onsite' }}" description="{{::$ctrl.parent.listing.Services_Onsite}}"
```

Comment:
In the my-favorites branch i've updated isOwnershipListing() to require you to pass in the listing.

so here we'd need $ctrl.parent.isOwnershipListing($ctrl.parent.listing)